web/src/app/users/UserContactMethodForm.js

```diff
@@ -52,40 +51,25 @@ export default class UserContactMethodForm extends React.PureComponent {
               <MenuItem value='VOICE'>VOICE</MenuItem>
             </FormField>
           </Grid>
-          <Grid item xs={12} sm={12} md={6}>
-            <TextField
-              fullWidth
-              name='countryCode'
-              label='Country Code'
-              value={getCountryCode(this.props.value.value)}
-              onChange={e =>
-                this.props.onChange({
-                  ...this.props.value,
-                  value:
-                    e.target.value + stripCountryCode(this.props.value.value),
-                })
-              }
-              select
-            >
-              <MenuItem value='+1'>+1 (USA)</MenuItem>
-              <MenuItem value='+91'>+91 (India)</MenuItem>
-              <MenuItem value='+44'>+44 (UK)</MenuItem>
-            </TextField>
-          </Grid>
-          <Grid item xs={12} sm={12} md={6}>
+          <Grid item xs={12} sm={12}>
             <FormField
+              placeholder={'+1 123-456-7891'}
```

Comment:
I don't think this would be valid input, do we need it?

Also, if using an example number use a fake one with 555 like `123-555-0123` so we don't embed someones actual number by random chance.